### DISCUSSION TITLE
The tab with the icon becomes smaller when the text is long

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#7677] [Help]
<img width="1292" alt="Screenshot 2023-08-11 at 14 14 25" src="https://github.com/filamentphp/filament/assets/121255432/0459fd18-5bdd-451e-ba28-dcbf16fc3ae7">

**Comment by pboivin** [comment]
Indeed. What would you like to see instead?

**Reply by danharrin** [reply]
Probably just needs flex-shrink-0 on the icon

**Reply by devhoussam1998** [reply]
yeah `flex-shrink-0` works

<img width="997" alt="Screenshot 2023-08-12 at 12 43 19" src="https://github.com/filamentphp/filament/assets/121255432/f9f7c704-7941-48ae-9216-88151041cf08">

**Reply by danharrin** [reply]
Please PR it in
